.github/workflows/prune.yml

```diff
@@ -0,0 +1,28 @@
+name: "Prune"
+
+on:
+  schedule:
+    # Runs at 1:22 PM / 13:22 in the UTC timezone which will run at 3:22 AM PST
+    # or 4:22 AM PDT (Daylight Savings). The GitHub Actions documentation
+    # recommends against running scheduled jobs at the top of the hour as they
+    # may be delayed or cancelled due to high loads from other customers'
+    # scheduled jobs.
+    - cron: "22 13 * * *"
+
+jobs:
+  packagecloud:
+    runs-on: ubuntu-24.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup go
+        uses: WillAbides/setup-go-faster@v1
+        with:
+          go-version: stable
+
+      - name: Run packagecloudpruner
```

Comment:
Isn't there a package retention policy that we could configure on packagecloud?

Reply:
Unfortunately not.